### ISSUE TITLE
Use CSS for styling

### ISSUE THREAD

**Issue by mybearworld** [GenericProgrammer1234/Intweb#1]
https://github.com/GenericProgrammer1234/Intweb/blob/bc5b4e4571d97dc347e5d8e429b5c5302ee8ba06/intweb/templates/intweb.html#L21-L22

That's not how `br` tags should be used.

**Comment by GenericProgrammer1234** [comment]
Why?

**Comment by mybearworld** [comment]
Semantic HTML. `<br>` tags specifically mean line breaks in paragraphs that do not warrant another paragraph.
The rule of thumb for deciding to use a semantic HTML tags (`<br>` in this case) or CSS (`margin-bottom` in this case) is: Use semantic HTML when removing the style would cause the content to have a different meaning.

So, for example, a poem would have `<br>` tags, as the line breaks are important for rhyming to look correct.
In this case, it's just for styling purposes, so it should be CSS.

See also the MDN article: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

> **Note**: Do not use `<br>` to create margins between paragraphs; wrap them in [`<p>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p) elements and use the [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) [`margin`](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) property to control their size.

> Creating separate paragraphs of text using `<br>` is not only bad practice, it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within `<br>`s. This can be a confusing and frustrating experience for the person using the screen reader.

**Comment by GenericProgrammer1234** [comment]
No I am using it to create space between the wrapper and the control buttons such as Add Element.

**Comment by mybearworld** [comment]
> No I am using it to create space between the wrapper and the control buttons such as Add Element.

And that has nothing to do with paragraphs, so `<br>` tags shouldn't be used.

**Comment by GenericProgrammer1234** [comment]
<br> tags are to create space so they should be used.

**Comment by mybearworld** [comment]
> `<br>` tags are to create space so they should be used.

No, `<br>` tags are not to create space. `<br>` tags are used to create new lines within paragraph. That's semantically what it's for. Nothing else. `margin`s are to create space.

**Comment by GenericProgrammer1234** [comment]
But `<br>` tags accomplish the same thing.

**Comment by mybearworld** [comment]
> But `<br>` tags accomplish the same thing.

Visually, yes. Semantically, no. If someone uses any other means of accessing the page than looking at it (screen readers, for instance), the information may be perceived incorrectly.

**Comment by GenericProgrammer1234** [comment]
> > But `<br>` tags accomplish the same thing.
> 
> Visually, yes. Semantically, no. If someone uses any other means of accessing the page than looking at it (screen readers, for instance), the information may be perceived incorrectly.

I think it would still be pretty obvious. How would they even know there are br tags, screen readers don't say "br tag br tag br tag br tag br tag br tag add element". They say "add element".

**Comment by mybearworld** [comment]
> I think it would still be pretty obvious. How would they even know there are br tags, screen readers don't say "br tag br tag br tag br tag br tag br tag add element". They say "add element".

Probably obvious, yes, but still not the best experience. It's not difficult to do this, and would improve the experience. Also yes, screen readers probably would not announce it like that, but they may pause a bit for every `<br>` tag, for example, which is good for paragraphs, but not for styling.

**Comment by GenericProgrammer1234** [comment]
Hmm.

**Comment by GenericProgrammer1234** [comment]
Your starting to convince me.

**Comment by GenericProgrammer1234** [comment]
Well I guess make a PR and implement it, I have more important things to implement.
